torch/_inductor/fx_passes/freezing_patterns.py

```diff
@@ -118,3 +123,141 @@ def addmm_fuse_replacement_second(inp, w1, w2, w3, b1, b2, b3):
         extra_check=check_concat_weights,
         exclusive_arg_names=("w1", "w2", "w3", "b1", "b2", "b3"),
     )
+
+
+@functools.lru_cache(None)
+def binary_folding_init():
+    _conv_args = [Arg() for _ in range(9)]
+    _computation_ops = [aten.convolution.default]
+    _binary_ops = [aten.add.Tensor, aten.sub.Tensor, aten.mul.Tensor, aten.div.Tensor]
+    _computation_calls = [CallFunction(aten.convolution.default, *_conv_args)]
+
+    def _is_constant_node(node):
```

Comment:
Yes, the checks are following it.

Reply:
Cool! Would you mind copying over some of the comments and reusing the tests ? And maybe moving to a separate file ?

I think it'll cool to remove the batch norm pass and replace it with this.

Reply:
Now, copy some of the comments and reuse the tests, for scalar case, I don't support it now, because it needs to use ```aten.full`` to create a new tensor, but it can't be folded(https://github.com/pytorch/pytorch/blob/main/torch/_inductor/freezing.py#L105). I will create another PR to remove the batch norm pass.